http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppRoutes.scala

```diff
@@ -167,6 +176,30 @@ class AppRoutes(kubernetesService: AppService[IO], userInfoDirectives: UserInfoD
       resp <- ctx.span.fold(apiCall)(span => spanResource[IO](span, "listApp").use(_ => apiCall))
     } yield StatusCodes.OK -> resp
 
+  private[api] def updateAppConfigHandler(userInfo: UserInfo,
+                                          googleProject: GoogleProject,
+                                          appName: AppName,
+                                          req: UpdateAppConfigRequest
+  )(implicit ev: Ask[IO, AppContext]): IO[ToResponseMarshallable] =
+    for {
+      _ <- foldSpan(kubernetesService.updateAppConfig(userInfo, CloudContext.Gcp(googleProject), appName, req),
+                    "updateAppConfig"
+      )
+    } yield StatusCodes.Accepted
+
+  // TODO: this pattern is repeated over 20x
```

Comment:
I'm not planning to commit foldSpan() as-is. Instead I would like a discussion about whether this sort of thing is desired.  Happy to remove if not.

Reply:
I personally like it, making the file DRYer while keeping the logic close enough for comprehension